src/systemctl/systemctl-logind.c

```diff
@@ -151,7 +151,9 @@ int logind_check_inhibitors(enum action a) {
         if (arg_transport != BUS_TRANSPORT_LOCAL)
                 return 0;
 
-        r = acquire_bus(BUS_FULL, &bus);
+        r = acquire_bus_full(BUS_FULL, /* graceful = */ true, &bus);
+        if (r == -ECONNREFUSED)
```

Comment:
This should also check uid == 0 to keep parity with the 256 behaviour. Also given we are past RC3, I'd much prefer if the fix for 257 was just the error check here, and the refactor was in a separate PR for 258

Reply:
OK. Done. PTAL!